src/Microsoft.ML.Tokenizers/Model/Model.cs

```diff
@@ -47,7 +43,23 @@ public virtual bool TokenizeToIds(string sequence, bool isSpecialToken, IList<in
             {
                 accumulatedIds.Add(token.Id);
             }
-            return true;
+        }
+
+        /// <summary>
+        /// Get the number of token's Ids that the input sequence will be encoded to.
+        /// </summary>
+        /// <param name="sequence">The text to tokenize.</param>
+        /// <param name="isSpecialToken">Indicate if the token is special token.</param>
+        /// <returns>The number of token's Ids that the input sequence will be encoded to.</returns>
+        /// <remarks>
+        /// This method does the default implementation that uses the TokenizeToIds method to get the number of token's Ids.
+        /// Tokenizer's models which care about performance may choose to override this method to provide a more efficient implementation.
+        /// </remarks>
+        public virtual int GetTokenizedIdsCount(string sequence, bool isSpecialToken)
```

Comment:
Same naming comment here... this isn't actually getting any "Ids"... can this just be CountTokens?

Reply:
Will use your suggestion.